lib/linguist/heuristics.yml

```diff
@@ -426,6 +426,11 @@ disambiguations:
     pattern: '^\s*(import.+(from\s+|require\()[''"]react|\/\/\/\s*<reference\s)'
   - language: XML
     pattern: '(?i:^\s*<\?xml\s+version)'
+- extensions: ['.vba']
+  rules:
+  - language: Vim script
+    pattern: '^UseVimball'
```

Comment:
Do you know if Vim scripts are invalid if they don't match this pattern?

Reply:
Se above. Vim scripts will be valid if they don't match this, but then they'll have `.vim` extension.

Reply:
Good. Let's keep the heuristic as it is then.